api-reference/docstubs/api/activitystatistics-list.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "List activityStatisticses"
+description: "List properties and relationships of the activityStatistics objects."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List activityStatisticses
+
+Namespace: microsoft.graph
+
+List properties and relationships of the [activityStatistics](../resources/activitystatistics.md) objects.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /activitystatistics
+GET /me/analytics/activityStatistics
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}.Required|
+
+## Request body
+Do not supply a request body for this method.
+
+## Response
+If successful, this method returns a `200 OK` response code and a collection of [activityStatistics](../resources/activitystatistics.md) objects in the response body.
+
+## Examples
+
+### Request
+Here is an example of the request.
+<!-- {
+  "blockType": "request",
+  "name": "get_activitystatistics"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/activitystatistics
```

Comment:
GLOBAL:
Perhaps we can add a placeholder alert for authors to review the generated examples, or better, add a real example?
Like:
"TO DO: Replace with real example. Generated example may have errors."

Reply:
Yeah - this is worth a discussion...